pkg/loop/standard_capabilities.go

```diff
@@ -69,6 +69,7 @@ type StandardCapabilities interface {
 		pipelineRunner core.PipelineRunnerService,
 		relayerSet core.RelayerSet,
 		oracleFactory core.OracleFactory,
+		gatewayConnector core.GatewayConnector,
```

Comment:
This change will need to be carried through to each capability. Is there a fast follow planned for that?

Reply:
yes can update all capabilities after this PR is merged